app/javascript/styles/routes/categories.scss

```diff
@@ -8,28 +8,27 @@
     .container { display: flex; }
 
     .scrollable-menus {
+      animation-duration: .8s;
+      animation-name: slidein;
+      background: $color_bg-light;
       flex: 1 0 29%;
-
-      max-width: 29%;
       margin-right: 2%;
+      max-width: 29%;
+      max-height: max-content;
 
-      background: $color_bg-light;
 
       &:nth-of-type(3) {
         flex: 1 0 38%;
-        max-width: 38%;
-
         margin-right: 0;
+        max-width: 38%;
       }
 
       h3 {
-        margin: 0;
-        padding: .5em 1.4em;
-
+        border: 1px solid $color_font-medium;
```

Comment:
Definitely think we should look to a BEM refactor of the h3 in the scrollable menus. I go back and forth as to whether it should be an h2 as well (they are the primary headers of the page aside from the DataCommon title and logo), but especially as we work towards standardizing the font styling system, directly referring to the tag as a child element is a bit confusing.

Reply:
Agreed, the CSS in here is a hot mess and probably could use much ❤️.